static/src/javascripts/projects/commercial/modules/header-bidding/utils.ts

```diff
@@ -176,17 +177,22 @@ export const shouldIncludeImproveDigitalSkin = (): boolean =>
  *
  * Add additional tests here when integrating a new SSP
  */
-const inSSPTest = (): boolean => isInABTestSynchronous(integrateCriteo);
+const inSSPTest = (): boolean =>
+	isInABTestSynchronous(integrateCriteo) ||
+	isInABTestSynchronous(integrateSmart);
 
 /**
- * Determine whether to include Criteo as a bidder
+ * Determine whether to include Criteo and Smart as bidders
  *
- * Include Criteo if visitor is not a participant in an AB test for integrating an SSP
- * or that they are in the variant of the Criteo test
+ * Include if visitor is not a participant in an AB test for integrating an SSP
+ * or if they are in the variant of their respective AB test
  */
 export const shouldIncludeCriteo = (): boolean =>
 	!inSSPTest() || isInVariantSynchronous(integrateCriteo, 'variant');
```

Comment:
This JSDoc only refers to `shouldIncludeCriteo`. It would be best to duplicate the relevant content to `shouldIncludeSmart` below to get information in your IDE.